packages/devtools_app/lib/src/framework/status_line.dart

```diff
@@ -186,7 +186,7 @@ class StatusLine extends StatelessWidget {
                           style: textTheme.bodyMedium,
                           overflow: TextOverflow.clip,
                         ),
-                      ]
```

Comment:
Example where a trailing comma doesn't change what dartfmt does but where it does seem fairly consistent with how we are using trailing commas.